translations/messages+intl-icu.en.xlf

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="nl" target-language="en" datatype="plaintext" original="file.ext">
+    <header>
+      <tool tool-id="symfony" tool-name="Symfony"/>
+    </header>
+    <body>
+      <trans-unit id="kCpMKtp" resname="Inloggen">
+        <source>Inloggen</source>
+        <target>Login</target>
+      </trans-unit>
+      <trans-unit id="WkaEMPh" resname="Sluiten">
+        <source>Sluiten</source>
+        <target>Close</target>
+      </trans-unit>
+      <trans-unit id="nWMiwfT" resname="Login">
+        <source>Login</source>
+        <target>Login</target>
+      </trans-unit>
+      <trans-unit id="Hl3PbWW" resname="U dient in te loggen om verder te gaan">
+        <source>U dient in te loggen om verder te gaan</source>
+        <target>You need to login to continue</target>
+      </trans-unit>
+      <trans-unit id="hZbA9KW" resname="C.S.R. Delft">
+        <source>C.S.R. Delft</source>
+        <target>C.S.R. Delft</target>
+      </trans-unit>
+      <trans-unit id="iXRdDVv" resname="De Civitas Studiosorum Reformatorum is een bruisende, actieve, christelijke studentenvereniging in&#10;&#9;&#9;&#9;&#9;&#9;&#9;Delft, rijk aan tradities die zijn ontstaan in haar %leeftijd%-jarig bestaan. Het is een&#10;&#9;&#9;&#9;&#9;&#9;&#9;breed gezelschap&#10;&#9;&#9;&#9;&#9;&#9;&#9;van zo'n 275 leden met een zeer gevarieerde (kerkelijke) achtergrond, maar met een duidelijke&#10;&#9;&#9;&#9;&#9;&#9;&#9;eenheid door het christelijk geloof. C.S.R. is de plek waar al tientallen jaren studenten goede&#10;&#9;&#9;&#9;&#9;&#9;&#9;vrienden van elkaar worden, op intellectueel en geestelijk gebied groeien en goede studentengrappen&#10;&#9;&#9;&#9;&#9;&#9;&#9;uithalen.">
+        <source>De Civitas Studiosorum Reformatorum is een bruisende, actieve, christelijke studentenvereniging in
+						Delft, rijk aan tradities die zijn ontstaan in haar %leeftijd%-jarig bestaan. Het is een
+						breed gezelschap
+						van zo'n 275 leden met een zeer gevarieerde (kerkelijke) achtergrond, maar met een duidelijke
+						eenheid door het christelijk geloof. C.S.R. is de plek waar al tientallen jaren studenten goede
+						vrienden van elkaar worden, op intellectueel en geestelijk gebied groeien en goede studentengrappen
+						uithalen.</source>
+        <target>Civitas Studiosorum Reformatorum is a vibrant, enthusiastic christian student association in Delft,
+        rich in traditions that have developed over the past %leeftijd% years.
+        We are counting 275 members coming from all kinds of church denominations, united in christian faith. C.S.R. is a community
```

Comment:
`church denominations` is hier denk ik niet juist. `Christian denominations` zou hier beter passen.

> ... all kinds of different churches, ...

Zou misschien ook wel prima werken.